src/Packaging/Packaging.csproj

```diff
@@ -134,6 +134,102 @@
     <Message Importance="high" Text="End downloading scanner cli" />
   </Target>
 
+  <UsingTask TaskName="ValidateZipFile" TaskFactory="CodeTaskFactory" AssemblyFile="$(MSBuildToolsPath)\Microsoft.Build.Tasks.v12.0.dll">
+    <ParameterGroup>
+      <FullFilePath ParameterType="System.String" Required="true" />
+    </ParameterGroup>
+    <Task>
+      <Code Type="Fragment" Language="cs">
+        <![CDATA[    
+        // Sanity check that the zip file has the expected header. This isn't a foolproof check, but
+        // it does catch errors encountered previously due differences between zipping/unzipping on
+        // different platforms.
+        
+        // See http://petlibrary.tripod.com/ZIP.HTM
+          byte[] expectedHeader = new byte[] { 0x50, 0x4B, 0x03, 0x04, 0x14, 0x00, 0x00, 0x08 };
```

Comment:
You are using the explicit type here and `var` on the next line, could you use one pattern or the other but not both?